test/hub/integration/Cases.t.sol

```diff
@@ -182,7 +182,7 @@ contract TestCases is BaseTest {
         uint128 poolDecimals = (10 ** hubRegistry.decimals(USD.raw())).toUint128();
         uint128 assetDecimals = (10 ** hubRegistry.decimals(USDC_C2.raw())).toUint128();
 
-        cv.updateHoldingAmount(poolId, scId, USDC_C2, 1000 * assetDecimals, D18.wrap(1e18), true);
+        cv.updateHoldingAmount(poolId, scId, USDC_C2, 1000 * assetDecimals, D18.wrap(1e18), true, true);
```

Comment:
NIT. (I can do this in a future PR, and also for other booleans). What about defining a `IS_SNAPSHOT` bool constant?